app/controllers/curation_concerns/generic_works_controller.rb

```diff
@@ -34,5 +34,11 @@ def t(key, interpolations = {})
     end
     helper HelperOverride
 
+    # Adds the 'My Works' breadcrumb; we only want this for logged-in users
+    # overrides https://github.com/samvera/sufia/blob/v7.3.1/app/controllers/concerns/sufia/works_controller_behavior.rb#L93
+    def add_breadcrumb_for_controller
+      super if current_ability.current_user.staff?
+    end
```

Comment:
`staff?` is the same as logged-in in our application. We wrote `staff?` to be a short way to check whether someone is logged in. A non-logged in user still has current_user because there's a dummy / non-persisted user that's used kind of as a nil object.

https://github.com/chemheritage/chf-sufia/blob/master/app/models/user.rb#L34